Source/relay/TourGuide/Items of the Month/2024/Chest Mimic

```diff
@@ -0,0 +1,26 @@
+//2024
+//Chest Mimic
+RegisterResourceGenerationFunction("IOTMChestMimicGenerateResource");
+void IOTMChestMimicGenerateResource(ChecklistEntry [int] resource_entries)
+{
+	if (!lookupFamiliar("Chest Mimic").familiar_is_usable()) return;
+
+	// Title
+    int famExperienceGain = numeric_modifier("familiar experience") +1;
+	int chestExperience = ($familiar[chest mimic].experience);
+	int famExpNeededForNextEgg = (50 - (chestExperience % 50));
+	int mimicEggsLeft = clampi(11 - get_property_int("_mimicEggsObtained"), 0, 11);
+	string [int] description;
+	string url = "familiar.php";
+	{
```

Comment:
No need for this extra set of braces, remove this one and the closing one below and de-dent the stuff inside them